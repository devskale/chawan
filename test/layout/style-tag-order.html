<!DOCTYPE html>
<style>
* { color: blue }
div { color: red }
body > span { color: green }
</style>
<style>
div { color: green }
span { color: red }
</style>
<div>test</div>
<span>test</span>
