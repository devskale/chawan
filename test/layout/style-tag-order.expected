[38;2;41;169;42mtest[39m
[38;2;41;169;42mtest[39m
